pom.xml

```diff
@@ -139,5 +139,28 @@
             <version>1.5.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
+            <version>1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>1.4</version>
```

Comment:
1.4.2 is the last release in the 1.580.x-compatible line.

BTW good style to extract the version into a POM property.